examples/chain_client/exchange/3_MsgInstantSpotMarketLaunch.py

```diff
@@ -35,14 +35,16 @@ async def main() -> None:
     await client.fetch_account(address.to_acc_bech32())
 
     # prepare tx msg
-    message = composer.msg_instant_spot_market_launch(
+    message = composer.msg_instant_spot_market_launch_v2(
         sender=address.to_acc_bech32(),
         ticker="INJ/USDC",
-        base_denom="INJ",
-        quote_denom="USDC",
+        base_denom="inj",
+        quote_denom="factory/inj17vytdwqczqz72j65saukplrktd4gyfme5agf6c/usdc",
```

Comment:
Why is it updated like that? Because it's the "1st level" msg constructor, that's why we use "precise" (in the case of a factory token - with `factory` prefix, in the case of the `ibc` token - with ibc prefix) token name rather than generic name?

Reply:
There is no longer internal translation from token symbol to token denom. In V2 the user is expected to provide the token denom